src/V3Timing.cpp

```diff
@@ -114,14 +115,17 @@ class TimingVisitor final : public VNVisitor {
     V3UniqueNames m_intraLsbNames{"__Vintralsb__"};  // Intra assign delay LSB var names
     V3UniqueNames m_forkNames{"__Vfork__"};  // Fork name generator
     V3UniqueNames m_trigSchedNames{"__VtrigSched"};  // Trigger scheduler name generator
+    V3UniqueNames m_dynTrigNames{"__VdynTrigger__"};  // Dynamic trigger name generator
```

Comment:
Should __VtrigSched also have a __ after it?

Reply:
That would be consistent with the other `V3UniqueNames` in this file. But I see there are different conventions throughout Verilator. Perhaps we should have `V3UniqueNames` automatically add a `__V` prefix and a `__` suffix?

Reply:
https://github.com/verilator/verilator/pull/3692